docs/design-deployment.rst

```diff
@@ -25,6 +25,7 @@ The components that make up mite are:
 - recorder*: logs messages appearing on the mite message bus to a file
 - collector: listens for special messages on the bus and records them
   to a file.  This is used for :ref:`data creation scenarios <data-creation-scenarios>`.
+- receiver*: dispatches incoming messages to connected ``processors``
```

Comment:
Doesn't need an asterisk bc it's not a singleton

Reply:
fixed and rearranged the components by layers, top to bottom and alphabetically per layer